public/tutorials/cross-dom-bridge-erc20.js

```
@@ -1,93 +1,185 @@
+  await publicClientL1.waitForTransactionReceipt({ hash: tx });
+  
+  const l1Balance = await publicClientL1.readContract({
+    address: l1Token,
+    abi: erc20ABI,
+    functionName: 'balanceOf',
+    args: [account.address]
+  });
+  console.log(`L1 Balance after receiving faucet: ${formatEther(l1Balance)}`);
+
+  console.log('Approving tokens for bridge...');
+  const bridgeAddress = optimismSepolia.contracts.l1StandardBridge[sepolia.id].address;
```

ESLint

[error] Extra semicolon.

(semi)